sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/configuration/AadResourceServerConfiguration.java

```diff
@@ -85,7 +85,6 @@ public JwtDecoder jwtDecoder(AadAuthenticationProperties aadAuthenticationProper
      * Default configuration class for using AAD authentication and authorization. User can write another configuration
      * bean to override it.
      */
-    @Configuration(proxyBeanMethods = false)
```

Comment:
why do we remove this here?

Reply:
@EnableWebSecurity is a derived annotation of @Configuration.